src/pages/euiseon/Login/Login.scss

```
@@ -0,0 +1,78 @@
+.wrap {
+  .container {
+    display: flex;
+    justify-content: center; /*space-evenly, space-around도 가운데로 정렬해줌*/
+    align-items: center;
+    height: 100%;
+    background: #dbdbdb;
+    .inner {
+      width: 400px;
+      height: 450px;
+      padding: 10px 40px;
+      box-sizing: border-box;
+      background: #fff;
+      border: 1px solid #dbdbdb;
+      .header {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin: 30px 0;
+
+        .logo {
+          font-family: 'Lobster', cursive;
+          font-size: 50px;
+        }
+      }
+
+      .form {
+        margin: 30px 0;
+
+        .input-box {
+          width: 100%;
+          height: 45px;
+          margin-bottom: 100px;
+          #userID,
+          #userPW {
+            margin-bottom: 10px;
+            padding: 10px;
+
+            input {
+              width: 100%;
+              height: 100%;
+              box-sizing: border-box;
+              background: #fafafa;
+              border: 1px solid #dbdbdb;
+              border-radius: 4px;
+              font-size: 15px;
+            }
+          }
+        }
```

- `input-box` -> `input_box` 오타 수정해주세요!
- 네스팅 순서 다시 확인해주세요! 현재 네스팅 순서의 의미는 .input_box 아래 #userID 아래 input 태그인데, 여기에 해당하는 태그는 없습니다!
```suggestion
      .form {
        margin: 30px 0;

          input {
            width: 100%;
            height: 100%;
            box-sizing: border-box;
            background: #fafafa;
            border: 1px solid #dbdbdb;
            border-radius: 4px;
            font-size: 15px;
            
            &.input_box {
              width: 100%;
              height: 45px;
              margin-bottom: 100px;
            }
          }
        }
```
- 위와 같은 느낌으로 네스팅 순서가 수정되어야 할 것 같습니다! 참고해서 수정해주세요!

---

> * `input-box` -> `input_box` 오타 수정해주세요!
> * 네스팅 순서 다시 확인해주세요! 현재 네스팅 순서의 의미는 .input_box 아래 #userID 아래 input 태그인데, 여기에 해당하는 태그는 없습니다!
> 
> * 위와 같은 느낌으로 네스팅 순서가 수정되어야 할 것 같습니다! 참고해서 수정해주세요!

넵 확인했습니다 수정 하겠습니다!